module/VuFind/src/VuFindTest/Feature/FixtureTrait.php

```diff
@@ -64,6 +86,7 @@ protected function getFixtureDir($module = 'VuFind')
      */
     protected function getFixture($filename, $module = 'VuFind')
     {
+        $realFilename = $this->getFixturePath($filename, $module);
         $realFilename = realpath($this->getFixtureDir($module) . $filename);
```

Comment:
Looks like you added a line here but didn't delete the old line, so there's double work being done.